starwars/src/components/Characters/Characters.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { Container, Button, Col, Row } from "reactstrap";
+import CharacterCard from "./CharacterCard";
+
+function Characters() {
+  const [people, setPeople] = useState([]);
+  const [page, setPage] = useState(1);
+
+  useEffect(() => {
+    axios
+      .get(`https://swapi.co/api/people/?format=json&page=${page}`)
+      .then(response => {
+        setPeople(response.data.results);
+      })
+      .catch(error => {});
```

Comment:
A better way to handle error is to create another slice of state, then render the error in the view
e.g
```
const [error, setError] = useState(null);
```
then in the catch block
```
 .catch(error => {
        setError(error.message);
      })
```
then in the jsx:
```
{ error && <p>{error}</p>